pages/common/mtr.md

```diff
@@ -1,28 +1,32 @@
 # mtr
 
 > Matt's Traceroute: combined traceroute and ping tool.
-> More information: <https://bitwizard.nl/mtr>.
+> More information: <https://www.bitwizard.nl/mtr/>.
```

Comment:
```suggestion
> More information: <https://bitwizard.nl/mtr/>.
```
The URL seems to work without `www` too so maybe we can skip its addition.

Reply:
On https://github.com/traviscross/mtr it is linked to www and since that is the official git repo, I took it that way.